src/vm/moar/QAST/QASTOperationsMAST.nqp

```diff
@@ -2422,7 +2422,7 @@ QAST::MASTOperations.add_core_moarop_mapping('isge_n', 'ge_n');
 
 QAST::MASTOperations.add_core_moarop_mapping('cmp_s', 'cmp_s');
 QAST::MASTOperations.add_core_moarop_mapping('unicmp_s', 'unicmp_s');
-QAST::MASTOperations.add_core_moarop_mapping('getstrfromname', 'getstrfromname');
+QAST::MASTOperations.add_core_moarop_mapping('strfromname', 'getstrfromname');
```

Comment:
Do we want to keep the old mapping and the new mapping? I found it being used in the ecosystem:

https://gist.github.com/Whateverable/3f2e643323a2ec54f7732abd114ebe5d

Reply:
We always say we don't make any promises about NQP (e.g., what ops there are, what they do), so say don't leave the old mapping and I'll just make a PR to that module after the rename happens.